core/templates/dev/head/pages/exploration_editor/history_tab/history_tab.html

```diff
@@ -31,6 +31,8 @@ <h3>List of Changes</h3>
         <[versionCountPrompt]>
       </div>
     </div>
+    <br>
+      <pagination boundary-links="true" class="pagination-sm" max-size="5" items-per-page="versionsPerPage" total-items="versionCheckboxArray.length" ng-model="currentPage" ng-change="pageChanged()"></pagination>
```

Comment:
Deindent by 2, since this isn't a child of br.

Reply:
Done.